x-pack/test_serverless/api_integration/test_suites/common/platform_security/api_keys.ts

```diff
@@ -54,23 +66,22 @@ export default function ({ getService }: FtrProviderContext) {
             role_descriptors: {},
           };
 
-          ({ body, status } = await supertest
+          ({ body, status } = await supertestWithoutAuth
             .post('/internal/security/api_key')
             .set(svlCommonApi.getCommonRequestHeader())
             .send(requestBody));
           // expect a rejection because we're not using the internal header
           expect(body).toEqual({
-            statusCode: 400,
-            error: 'Bad Request',
-            message: expect.stringContaining(
-              'method [post] exists but is not available with the current configuration'
-            ),
+            statusCode: 401,
+            error: 'Unauthorized',
```

Comment:
That is interesting. _question_ is it expected for operator user to get 400 Bad request?